app/src/components/common/SourceUrl/index.tsx

```diff
@@ -48,13 +51,16 @@ export const SourceConditionInput: React.FC<SourceProps> = ({ source, onSourceCh
         </Select>
       </Col>
       <Input
+        autoFocus
```

Comment:
It is not necessary for the input in this component to be by default focussed for other instances we use or may use this component in  (eg. test url & rule editors ).
Can we make it auto focus by passing a prop to this component?